src/models/lucky_cli_version.cr

```diff
@@ -6,6 +6,6 @@ module LuckyCliVersion
   end
 
   def current_version : String
-    "0.18.3"
+    "0.19.0"
```

Comment:
Weird... I thought I did this in my upgrade... 🤔

Reply:
https://github.com/luckyframework/website/pull/217/files#diff-f237c42c10af8be1e5d4906046b88605R9